src/components/security_manager/src/crypto_manager_impl.cc

```diff
@@ -283,6 +283,9 @@ bool CryptoManagerImpl::Init() {
 
 bool CryptoManagerImpl::OnCertificateUpdated(const std::string& data) {
   LOG4CXX_AUTO_TRACE(logger_);
+  sync_primitives::AutoLock lock(crypto_manager_lock_);
+  LOG4CXX_DEBUG(logger_,
+                "CryptoManager is locked. Start of sertificate update");
```

Comment:
@SKobziev This logs are redundant. Please Just swap Lock and autotrace

Reply:
@LuxoftAKutsan removed redundant logs [160fe94d](https://github.com/smartdevicelink/sdl_core/pull/2777/commits/160fe94dbd79df9521be91fea5f57551a896f607)